app/webpacker/lib/requests/routes.js.erb

```diff
@@ -97,7 +97,7 @@ export const adminFixResultsUrl = (personId, competition_id = undefined, event_i
 
 export const paymentFinishUrl = (competitionId, paymentIntegration) => `<%= CGI.unescape(Rails.application.routes.url_helpers.registration_payment_completion_url("${competitionId}", "${paymentIntegration}", host: EnvConfig.ROOT_URL)) %>`
 
-export const availableRefundsUrl = (competitionId, userId) => `<%= CGI.unescape(Rails.application.routes.url_helpers.refunds_path) %>?attendee_id=${competitionId}-${userId}`
+export const registrationPaymentsUrl = (competitionId, registrationId) => `<%= CGI.unescape(Rails.application.routes.url_helpers.competition_registration_payments_path("${competitionId}", "${registrationId}")) %>`
```

Comment:
If I'm reading the `routes.rb` correctly, then `competitionId` is not even needed as a route parameter anymore

Reply:
it is, it's under the   `resources :competitions do`

Reply:
Oh wait, that trick only worked for `resources :registrations` because it's marked as `shallow: true`. In that case, I think you should really consider my other comment above about nesting the route, because otherwise supplying `competitionId` *and* `registrationId` (which implicitly also identifies the competition) is kinda redundant. Nesting would make this route become simpler and profit from the `shallow` routing as well

Reply:
ok, I'll nest it